ipatests/prci_definitions/nightly_latest_389ds.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         git_refspec: '{git_refspec}'
         template: &389ds-master-latest
           name: freeipa/389ds-master-f30
-          version: 0.0.1
+          version: 0.0.2
```

Comment:
The box name is freeipa/389ds-master-f30 but I would expect it to be based on f31. Either the name is misleading or the template is using the wrong fedora version.

Reply:
@flo-renaud, thanks for looking so deep and finding that detail! :-)

Actually it's because copr repo [@389ds/389-ds-base-nightly](https://copr.fedorainfracloud.org/coprs/g/389ds/389-ds-base-nightly/) doesn't provide packages for Fedora 31, given the main purpose of this build is to test packages from that repo I kept it in Fedora 30. But you're right, the name is misleading, for this case "latest" is out-dated.

I not sure how to proceed, if I revert the renaming for this case I'll need to re-do it when those packages become available, if I leave the way it is at the moment it wouldn't reflect the reality of fedora releases.

The commit message tries to describe the that, probably not clear enough.

Reply:
Hi @netoarmando 
I just sent an email to 389ds team asking them to provide a nightly build of 389ds on fedora 31. If you prefer not to wait for the build, I'm OK with this PR, provided a ticket is opened to remember switching to a new box as soon as the f31 nightly build of 389ds is available. In this case you can keep the naming
```
         template: &389ds-master-latest
          name: freeipa/389ds-master-f30
```
Thanks.

Reply:
Template changed to `freeipa/389ds-master-f31`, copr repo were updated.